drivers/rawexec/driver_test.go

```diff
@@ -123,6 +123,15 @@ func TestRawExecDriver_SetConfig(t *testing.T) {
 	bconfig.PluginConfig = data
 	require.NoError(harness.SetConfig(bconfig))
 	require.Exactly(config, d.(*Driver).config)
+
+	// Turns on uid/gid restrictions, and sets the range to a bad value
+	config.DeniedHostUids = "10-0"
+	data = []byte{}
+	require.NoError(basePlug.MsgPackEncode(&data, config))
```

Comment:
We're trying to get everything of testify

```suggestion
	must.NoError(basePlug.MsgPackEncode(&data, config))
```

(Apply to the rest of the tests in the PR as needed)